src/code/bank0.asm

```diff
@@ -1855,57 +1855,65 @@ CheckItemsToUse::
     ; if shield is not equiped in slot B
     ld   a, [wBButtonSlot]                        ; $1235: $FA $00 $DB
     cp   INVENTORY_SHIELD                         ; $1238: $FE $04
-    jr   nz, .nextItemB                           ; $123A: $20 $0F
+    jr   nz, .otherItemB                           ; $123A: $20 $0F
     ; update shield status
     ld   a, [wShieldLevel]                        ; $123C: $FA $44 $DB
     ld   [wHasMirrorShield], a                    ; $123F: $EA $5A $C1
     ; reset shield if button not longer pressed down
     ldh  a, [hPressedButtonsMask]                 ; $1242: $F0 $CB
     and  J_B                                      ; $1244: $E6 $20
-    jr   z, .nextItemB                            ; $1246: $28 $03
+    jr   z, .otherItemB                            ; $1246: $28 $03
     ; the two checks from A does not apear here == bug?
     ; use the shield
     call SetShieldVals                            ; $1248: $CD $40 $13
 
-.nextItemB
+.otherItemB
+    ; skip if button is not pressed
     ldh  a, [hJoypadState]                        ; $124B: $F0 $CC
     and  J_B                                      ; $124D: $E6 $20
-    jr   z, .jr_125E                              ; $124F: $28 $0D
+    jr   z, .otherItemA                              ; $124F: $28 $0D
     ld   a, [$C1AD]                               ; $1251: $FA $AD $C1
     cp   $02                                      ; $1254: $FE $02
-    jr   z, .jr_125E                              ; $1256: $28 $06
+    jr   z, .otherItemA                              ; $1256: $28 $06
 
     ; Use item in B slot
     ld   a, [wBButtonSlot]                        ; $1258: $FA $00 $DB
     call UseItem                                  ; $125B: $CD $9C $12
 
-.jr_125E
+.otherItemA
+    ; skip if button is not pressed
     ldh  a, [hJoypadState]                        ; $125E: $F0 $CC
     and  J_A                                      ; $1260: $E6 $10
-    jr   z, .jr_1275                              ; $1262: $28 $11
+    jr   z, .SwordB                               ; $1262: $28 $11
     ld   a, [$C1AD]                               ; $1264: $FA $AD $C1
     cp   $01                                      ; $1267: $FE $01
-    jr   z, .jr_1275                              ; $1269: $28 $0A
+    jr   z, .SwordB                               ; $1269: $28 $0A
     cp   $02                                      ; $126B: $FE $02
-    jr   z, .jr_1275                              ; $126D: $28 $06
+    jr   z, .SwordB                               ; $126D: $28 $06
 
     ; Use item in A slot
     ld   a, [wAButtonSlot]                        ; $126F: $FA $01 $DB
     call UseItem                                  ; $1272: $CD $9C $12
 
-.jr_1275
+.SwordB
```

Comment:
Nitpick: local labels (prefixed with a dot) usually tend to have their first letter in lower-case. This is not strictly enforced everywhere, but it helps making local and global labels visually different.

Reply:
I will make a next pull request for that.
Could you confirm and close this one?
Screwed it up up my side.